ChameleonForms/Attributes/RequiredFlagsEnumAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ChameleonForms.Attributes
+{
+    /// <summary>
+    /// Marks a Flags enum property as required.
+    /// </summary>
+    public class RequiredFlagsEnumAttribute : RequiredAttribute
+    {
+        /// <inheritdoc />
+        public override bool IsValid(object value)
+        {
+            if (Convert.ToInt64(value) == 0L)

```

Comment:
Yep, but the model binder will act first so as long as someone doesn't apply this attribute to something that isn't a flag enum it should be OK.